src/actions/getPOI.action.js

```diff
@@ -0,0 +1,46 @@
+import * as actionTypes from './actionTypes'

```

Comment:
sorry if the task was unclear, but this PR should implement the redux stuff for all existing POI endpoints. so the POI section in the [apiWrapper](https://github.com/hack4impact-uiuc/nnb-frontend/blob/master/src/utils/apiWrapper.js#L51). 

some of the endpoints are different than the v2 endpoints, but we cant change that until we deploy the new api